ts/components/animations/ShakeAnimation.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+import { Animated, Easing } from "react-native";
+
+type Props = Readonly<{}>;
```

Comment:
Not a big problem but Props are already restricted to "Readonly" by the React.PureComponent typing. Non need to add Readonly here.

Reply:
If you want you can add "duration" as prop :)

Reply:
yes I second adding the `duration` as prop to parametrize the duration of the delay